File: 3232-find-if-digit-game-can-be-won/3232-find-if-digit-game-can-be-won.cpp
class Solution {
public:
    bool canAliceWin(vector<int>& nums) {
         int sd = 0 , ld = 0;
        for(int num : nums){
            if(num <= 9){
                sd += num;
            }
            else{
                ld += num;
            }
        }
        return sd != ld;
    }
    
};

File: 3232-find-if-digit-game-can-be-won/3232-find-if-digit-game-can-be-won.java
class Solution {
    public boolean canAliceWin(int[] nums) {
        int sd = 0 , ld = 0;
        for(int i = 0 ; i < nums.length ; i++){
            if(nums[i] <= 9){
                sd += nums[i];
            }
            else{
                ld += nums[i];
            }
        }
        return sd != ld;
    }
}